examples/cra-kitchen-sink/src/components/FlowTypeButton.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
```

Comment:
seems a bit much to add rules for flow just to support an example file but either way, I'll take it.